R/data-storage.R

```diff
@@ -91,6 +97,18 @@ DataStorage <- R6::R6Class( # nolint object_name_linter
   private = list(
     .username = NULL,
     .session_id = NULL,
+    check_date = function(date_value, .var_name) {
```

Comment:
function that checks date from `read_*_data` and allows for strings as long as they are accepted as dates